tests/Controller/DemoControllerTest.php

```diff
@@ -71,6 +71,11 @@ public static function provideTestFormSubmitData(): Iterator
             'rector_config' => 'PHP config should not include func call',
         ]];
 
+        // Add no rule in config
+        yield ['<?php echo "test"; ?>', '<?php $rectorConfig->removeUnusedImports(); ?>', [
+            'rector_config' => 'PHP config should include at least 1 rector rule',
```

Comment:
maybe instead of "bugging" the user with a error, would it be possible to add a noop rule automatically?

Reply:
this is the simplest solution I can found right now, add new rule in the config seems requires more complex refactor to detect type and position.

Reply:
thank you!